src/Input.h

```diff
@@ -194,6 +194,9 @@ class Input::Manager {
 	bool IsMouseYInvert() { return mouseYInvert; }
 	void SetMouseYInvert(bool state);
 
+	bool IsMiddleMouseButton() { return noMiddleMouseButton; }
+	void SetMiddleMouseButton(bool state);
```

Comment:
Recommend this be `EmulateMiddleMouseButton` instead of `IsMiddleMouseButton`, currently it reads as though the method is querying whether the middle mouse button is down.

Reply:
Fixed in 4b49af24bd2dcea54f889c9dcd9f0c83ab3f9f2b